packages/protocol/blockchain_scripts/deployHelpers/index.ts

```diff
@@ -60,10 +78,47 @@ const MAINNET_CHAIN_ID = "1"
 type MainnetChainId = typeof MAINNET_CHAIN_ID
 const RINKEBY_CHAIN_ID = "4"
 type RinkebyChainId = typeof RINKEBY_CHAIN_ID
-
-export type ChainId = LocalChainId | RopstenChainId | MainnetChainId | RinkebyChainId
-
-const CHAIN_IDS = genExhaustiveTuple<ChainId>()(LOCAL_CHAIN_ID, ROPSTEN_CHAIN_ID, MAINNET_CHAIN_ID, RINKEBY_CHAIN_ID)
+const FUJI_CHAIN_ID = "43113"
```

Comment:
The DAO has not approved any cross-chain deployments of Goldfinch. Goldfinch is only on ethereum mainnet.

These chain id changes will not get merged.

Reply:
these changes shouldn't have been pushed. will revert and keep only the relevant ones